File: src/main/java/um/haberes/report/model/tesoreria/core/CursoCargoContratadoDto.java
package um.haberes.report.model.tesoreria.core;

import java.math.BigDecimal;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import um.haberes.report.model.haberes.core.ContratadoPersonaDto;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CursoCargoContratadoDto {

    private Long cursoCargoContratadoId;

    private Long cursoId;

    private int anho = 0;

    private int mes = 0;

    private Long contratadoId;

    private Long contratoId;

    private Integer cargoTipoId;

    private BigDecimal horasSemanales = BigDecimal.ZERO;

    private BigDecimal horasTotales = BigDecimal.ZERO;

    private Integer designacionTipoId;

    private Integer categoriaId;

    private Long cursoCargoNovedadId;

    private byte acreditado = 0;

    private ContratadoPersonaDto contratadoPersona;

}

File: src/main/java/um/haberes/report/kotlin/dto/tesoreria/core/CursoCargoContratadoDto.kt
package um.haberes.report.kotlin.dto.tesoreria.core

import um.haberes.report.kotlin.dto.haberes.core.ContratadoPersonaDto
import java.math.BigDecimal

data class CursoCargoContratadoDto(

    var cursoCargoContratadoId: Long? = null,
    var cursoId: Long? = null,
    var anho: Int = 0,
    var mes: Int = 0,
    var contratadoId: Long? = null,
    var contratoId: Long? = null,
    var cargoTipoId: Int? = null,
    var horasSemanales: BigDecimal = BigDecimal.ZERO,
    var horasTotales: BigDecimal = BigDecimal.ZERO,
    var designacionTipoId: Int? = null,
    var categoriaId: Int? = null,
    var cursoCargoNovedadId: Long? = null,
    var acreditado: Byte = 0,
    var contratadoPersona: ContratadoPersonaDto? = null

)
